src/components/MatchProfilesForm/MatchProfilesForm.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```

Comment:
Add a note to the Changelog file.

Reply:
@VictorSoroka1 Thanks! Updated changelog file